pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -57,7 +80,9 @@ func (s *preFilterState) Clone() framework.StateData {
 		// Constraints are shared because they don't change.
 		Constraints:          s.Constraints,
 		TpKeyToCriticalPaths: make(map[string]*criticalPaths, len(s.TpKeyToCriticalPaths)),
-		TpPairToMatchNum:     make(map[topologyPair]int, len(s.TpPairToMatchNum)),
+		// The number of domains should not be changed by AddPod/RemovePod methods on PreFilter Extensions.
```

Comment:
`The number of domains does not change as a result of AddPod/RemovePod methods on PreFilter Extensions`